fuel-vm/src/util.rs

```diff
@@ -19,7 +20,7 @@
 /// let call = Call::new(contract_id, 0, 0).to_bytes();
 /// let asset_id = [0x00; 32];
 /// let transfer_amount: Word = 100;
-/// let gas_to_forward = 1_000_000;
+/// let gas_to_forward = 100_000;
```

Comment:
The original value specified here was out-of-range of the possible 18-bit values for the immediate (max 262,144), caught by the new instruction constructor.